app/Observers/UserObserver.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Observers;
+
+use App\Models\User;
+
+class UserObserver
+{
+    /**
+     * Handle the User "saved" event.
+     *
+     * @param  \App\Models\User  $user
+     * @return void
+     */
+    public function saved(User $user)
+    {
+       $changes = $user->getChanges(); 
+      unset($changes['updated_at']); 
+        //for calling out and logging important changes separately
+        foreach ($changes as $key => $value) {
```

Comment:
@snipe This looks a little gross, but it gives us the ability to separate all changes on a user and log them differently (for instance, a permissions change which may need to be called out differently) - instead of one json object per save. I _could_ do this on the actual Loggable method, but I feel like this lets that remain a little more pure and this can be abstracted up. Wanted to check before continuing with that idea.